projects/tags-input-demo/src/app/app.component.html

```diff
@@ -312,15 +315,40 @@
         [dropDownTemplate]="ddNestedTemplate"
         >
       </ti-angular-tags-input>
+      </div>
+
+      <div>
+        <h3 class="main__form__tt">Simple Dropdown</h3>
       <ti-angular-tags-input formControlName="users" class="main__form__ti"
         [tagsData]="sampleDataSimple"
         (tagAdded)="onTagAdded($event)"
         (valueChanged)="onValChanged($event)"
         [config]="simpleTagsInputConfig"
         >
       </ti-angular-tags-input>
+      </div>
+
+      <div>
+      <h3 class="main__form__tt">Image Dropdown</h3>
+      <ti-angular-tags-input formControlName="users" class="main__form__ti"
+      [tagsData]="sampleDataImage"
+      (tagAdded)="onTagAdded($event)"
+      (valueChanged)="onValChanged($event)"
+      [config]="ImageTagsInputConfig"
+      [dropDownTemplate]="userTagItemTemplate"
+      >
+    </ti-angular-tags-input>
+    </div>
+      
+
     </form>
   </div>
+
+  <ng-template #userTagItemTemplate let-item="item" let-tagClass="tagClass" let-config="config" let-closeClicked="closeClicked">
```

Comment:
We're using ng-template here also some `let-xyz` stuff. But we're not doing anything with those variables. For instance, we don't pass anything to `<tid-users-dropdown-template>`

So instead of having a component that has *ngFor* inside, we could have a component that is for a single item in the array.
Something like

```html
<ng-template let-items="items" let-config="config">
  <tid-user-list-item *ngFor="let item of items" [item]="item" [config]="config"></tid-user-list-item>
</ng-template>
```

Reply:
the component name needs to be changed to `tid-user-list-item`?